### DISCUSSION TITLE
PC remote Android cannot input Chinese

### DISCUSSION THREAD

**Discussion by haisong66** [rustdesk/rustdesk#8072] [General]
### Bug Description

PC remote Android cannot input Chinese

### How to Reproduce

PC can only input English to Android, other languages ​​​​are not supported

### Expected Behavior

I hope that the PC input method can be input directly into Android, or that the hardware keyboard can control the Android virtual keyboard

### Operating system(s) on local side and remote side

Windows10->1.2.4

### RustDesk Version(s) on local side and remote side

1.2.4->1.2.4

### Screenshots

https://github.com/rustdesk/rustdesk/assets/168391999/e9d133ff-fa1e-4b83-92ed-24734e0f9b92

**Comment by Korb** [comment]
A similar problem is the impossibility of entering something other than English when connecting from a Windows desktop to an Android smartphone. At the same time, both the PC and the smartphone have a Russian-language keyboard layout. But even if the Russian-language layout is selected on both the PC and the smartphone, the text entered is still in English.

---

Rustdesk for Android 1.3.7 (F-Droid)
Rustdesk for Windows 1.3.8 (Github)

OS version: Funtouch OS 14
Model: Bivo T1 (4G) / V2153
Hardware version: MP_0.1
Build number: PD22010F_EX_A_14.2.12.0.W20
Baseband version: .HA.1.1.c2-00054-DIVAR_GENSP_PACK-1.29062.107
Kernel version: 4.19.157-perf+, 1 Wed Jan 8 18:59:53 CST 2025
Compile time: 8 Jan 2025 13:56:50
Android security update: 1 January 2025
Google Play system update: 1 October 2024
Software parameter update: 7 January 2025

Microsoft Windows 10.0.19045.5487

---

Upd. The problem of impossibility of input on non-English keyboard layout was solved by changing `Input source 1` to `Input source 2`:

![rustdesk_v3ihB2Lzg6](https://github.com/user-attachments/assets/4a5e1f50-6fcf-44af-aa5e-e52efe9dcba9)

**Comment by Dai-StephenJose** [comment]
I tried to modify the input source according to your method, but it seems that I still cannot input Chinese.
